react/features/invite/components/AddPeopleDialog.web.js

```diff
@@ -189,6 +249,42 @@ class AddPeopleDialog extends Component<*, *> {
             || this.state.addToCallInProgress;
     }
 
+    _isMaybeAPhoneNumber: (string) => boolean;
+
+    /**
+     * Checks whether a string looks like it could be for a phone number.
+     *
+     * @param {string} text - The text to check whether or not it could be a
+     * phone number.
+     * @private
+     * @returns {boolean} True if the string looks like it could be a phone
+     * number.
+     */
+    _isMaybeAPhoneNumber(text) {
+        const alphabet = /[a-zA-Z]+/;
+        const digits = this._getDigitsOnly(text);
+
+        return Boolean(!alphabet.test(text) && digits.length);
```

Comment:
this feels like a weird middle ground to me.  we make sure it has at least one digit and no alphabet characters, but don't check for other invalid characters (like !#@$!#$).  but if the failure mode for a bad number isn't too bad maybe this is ok for now.